packages/UI/src/components/atoms/NewCloseButton/NewCloseButton.tsx

```diff
@@ -0,0 +1,28 @@
+import { tv, type VariantProps } from "tailwind-variants";
+
+const button = tv({
+    base: "bg-black shadow rounded  flex flex-row items-center justify-center",
+    variants: {
+      color: {
+        primary: "bg-white",
+        secondary: "bg-purple-500",
+       
+      },
+      size: {
+        sm: "p-2 text-xs",
+        md: "py-2 px-4 text-sm",
+        lg: "py-2 px-6 text-md",
+      },
+    },
+  });
+
+interface ButtonProps extends VariantProps<typeof button> {
```

Comment:
let's add all the types related to a button rather than just the custom ones we made, 

`interface ButtonProps extends VariantProps<typeof button>, React.DetailedHTMLProps<
      React.ButtonHTMLAttributes<HTMLButtonElement>,
      HTMLButtonElement
    >`